packages/bpk-docs/src/components/DocsPageBuilder/ComponentScreenshots.js

```diff
@@ -49,7 +49,9 @@ const ComponentScreenshot = ({ title, subText, ...rest }) => {
       className={getClassName('bpkdocs-component-screenshots__item')}
     >
       <dt className={getClassName('bpkdocs-component-screenshots__item-title')}>
-        <BpkText tagName="h3">{title}</BpkText>
+        <BpkText tagName="h3" textStyle="xs">
```

Comment:
Agreed with Will to do this as the blurb was using the same text style as the screenshot title and it looked weird in this PR.